src/Mono.Android/Android.Content.PM/PackageManager.cs

```diff
@@ -0,0 +1,79 @@
+namespace Android.Content.PM;
+
+#if ANDROID_34
+public abstract partial class PackageManager
+{
+	public sealed partial class PackageInfoFlags
+	{
+		// Create overloads that accept PackageInfoLongFlags
+		[global::System.Runtime.Versioning.SupportedOSPlatformAttribute ("android33.0")]
+		public static unsafe Android.Content.PM.PackageManager.PackageInfoFlags Of (PackageInfoLongFlags value)
+			=> Of ((long) value);
+
+		[global::System.Runtime.Versioning.SupportedOSPlatformAttribute ("android33.0")]
+		public unsafe PackageInfoLongFlags ValueAsFlags
+			=> (PackageInfoLongFlags) Value;
+	}
+}
+
+// Manually created "long" version of "PackageInfoFlags" enum, created from documentation:
+// https://developer.android.com/reference/android/content/pm/PackageManager.PackageInfoFlags#of(long)
+[System.Flags]
+public enum PackageInfoLongFlags : long
+{
+	None = 0,
+
+	GetActivities = 1,
```

Comment:
Two thoughts/questions:

 1. Instead of hardcoding the values ("from documentation"), could this instead re-use our existing enum for existing non-`long` values?

    ```csharp
    public enum PackageInfoFlagsLong : long {
        GetActivities = PackageInfoFlags.Activitites,
        GetServices = PackageInfoFlags.Services,
        // …
    }
    ```

 2. Why the "naming change" between `PackageInfoFlags` and `PackageInfoLongFlags`?  `PackageInfoFlags.Activities` is `PackageInfoLongFlags.GetActivities`, etc.  On the one hand this "new" convention may be "better", but it also means that migrating from the old enum to the new enum will require additional edits; you can't "*just*" change the enum name.